tests/validate/aie_pl_test/src/pl_controller.hpp

```diff
@@ -0,0 +1,219 @@
+/**
+* Copyright (C) 2019-2021 Xilinx, Inc
+*
+* Licensed under the Apache License, Version 2.0 (the "License"). You may
+* not use this file except in compliance with the License. A copy of the
+* License is located at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations
+* under the License.
+*/
+
+#pragma once
+#ifndef _XF_PLCTRL_PL_CONTROLLER_HPP_
+#define _XF_PLCTRL_PL_CONTROLLER_HPP_
+
+//#include "xclbin.h"
+#include <algorithm>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include <stdexcept>
+
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/property_tree/ptree.hpp>
+using boost::property_tree::ptree;
```

Comment:
Please remove the using.  In this case, the namespace should be used in the variable's declaration.
```suggestion

```